packages/api/src/services/security/authentication.service.ts

```diff
@@ -58,14 +60,55 @@ export class AuthenticationService implements Service {
   }
 
   /**
-   * Checks if security is enabled.
+   * Checks if security is enabled
    * @returns {boolean} True if security is enabled, false otherwise.
    */
   isSecurityEnabled(): boolean {
     return !!this.getSecurityConfig()?.enabled;
   }
 
+  canReadRepo(repository?: Repository): boolean {
```

Comment:
I'm not convinced this method should be here in the authentication service. Why did you decide to couple this service with some repository stuff?